readme.md

```diff
@@ -151,8 +151,13 @@ Tokens
 
 ### Creating a token
 
-A token can be used anywhere on Stripe where you would normally pass a card. Once it's created, it can be used on a
-customer or a charge, but only used once.
+A token can be used anywhere on Stripe where you would normally pass a card.
+Once it's created, it can be used on a customer or a charge, but only used once.
+
+In production, you'll want to create tokens with either
+[stripe.js](https://stripe.com/docs/tutorials/forms) or
+[Checkout](https://stripe.com/docs/tutorials/checkout), but it can be useful
+to create tokens with Stripe.net for testing.
```

Comment:
Related to my other comment, I'd be ok to soften the wording here somewhat, but I still think it should be made clear that the vast majority of users will want to create tokens with stripe.js / checkout.